File: 프로그래머스/0/181851. 전국 대회 선발 고사/전국 대회 선발 고사.js
// 참여 안하는사람 거르고 점수순으로 정렬
function solution(rank, attendance) {
  var answer = [];
  rank.forEach((score, i) => {
    if(attendance[i]) answer.push([score, i]) 
  })

  answer.sort((a, b) => {
    if(a[0] > b[0]) return 1;
    if(a[0] < b[0]) return -1
  })

  return 10000*answer[0][1] + 100*answer[1][1] + answer[2][1];
}

File: 프로그래머스/0/181851. 전국 대회 선발 고사/전국 대회 선발 고사.py
def solution(rank, attendance):
  answer = 0
  result = []

  for i, score in enumerate(rank):
    if attendance[i]:
      result.append([score, i])
    
  result.sort(key = lambda x: x[0])
  return 10000*result[0][1] + 100*result[1][1] + result[2][1]